src/rest-server/server/server.ts

```diff
@@ -1,11 +1,14 @@
-import express, { Router, Express } from "express";
+import express, { Express } from "express";
+import routes from "../routes";
 
 export class Server {
   private static instance: Server;
   private express: Express;
 
-  constructor() {
+  private constructor(private port?: number | string) {
     this.express = express();
+    this.settings();
+    this.routes();
```

Comment:
```suggestion
  private constructor() {
    this.express = express();
```

Con los cambios que te comento mas abajo el constructor quedaría asi